lib/kitchen/document.rb

```diff
@@ -44,7 +44,7 @@ def initialize(nokogiri_document:, config: nil)
       @nokogiri_document = nokogiri_document
       @location = nil
       @config = config || Config.new
-      @next_paste_count_for_id = {}
+      @id_cp_count = Hash.new { |hash, key| hash[key] = {} }
```

Comment:
If you change this to 

```ruby
@id_cp_count = Hash.new { |hash, key| hash[key] = {count: 0, last_paste: false} }
```

it'll give you the initial values you want, lets you skip some lines below